cfpb/consumerfinance.gov
# Move chart and simple-chart css to own files

Currently, a bunch of chart styles are being built into our `main.css` file needlessly.

Now that https://github.com/cfpb/consumerfinance.gov/pull/7055 is in, this pulls out those styles into their own separate files.

Shaves ~82kb off our ungzipped `main.css` file

Testing:
 - Pull and build
 - Visit http://localhost:8000/enforcement/enforcement-by-the-numbers/
 - Visit http://localhost:8000/data-research/consumer-credit-trends/auto-loans/origination-activity/
 - Everything should behave and function identically, except these pages should pull in `simple-chart.css` and `chart.css` respectively.
